src/main/kotlin/com/github/lppedd/cc/vcs/InternalVcsService.kt

```diff
@@ -181,24 +163,6 @@ internal class InternalVcsService(private val project: Project) : VcsService {
     return commitsMetadata
   }
 
-  @Suppress("unused")
-  @Compatibility(minVersion = "203.3645.34")
-  private fun ensureLogCreated(): Boolean {
-    val method =
-      VcsProjectLog::class.java.getDeclaredMethod("ensureLogCreated", Project::class.java)   // IDEA 203.3645.34+
-      ?: VcsProjectLog::class.java.getDeclaredMethod("getOrCreateLog", Project::class.java)  // IDEA 201.3803.32+
-      ?: VcsLogContentUtil::class.java.getDeclaredMethod("getOrCreateLog", Project::class.java)
-      ?: return false
-
-    return try {
-      method.isAccessible = true
-      method.invoke(null, project)
-      true
-    } catch (_: Exception) {
-      false
-    }
-  }
```

Comment:
I noticed the `@Suppress("unused")` but this was not used anywhere.

Now `com.intellij.vcs.log.impl.VcsLogContentUtil#getOrCreateLog(Project)` is deprecated as well.

If this need to be upgraded more recent documentation suggest to use `com.intellij.vcs.log.impl.VcsProjectLog#runWhenLogIsReady(Project, Consumer)` instead.